config/calico_versions.yml

```diff
@@ -0,0 +1,11 @@
+components:
+  typha:
+    version: v3.11.1
+  calico/node:
+    version: v3.11.1-with-auto-backend
+  calico/cni:
+    version: v3.11.1
+  calico/kube-controllers:
+    version: v3.11.1
+  flexvol:
+    version: v3.11.1
```

Comment:
I think we're using `v3.12.0` now

Reply:
This PR does not update the versions of any components. The current code suggests that we're still using flexvol `v3.11.1`: https://github.com/tigera/operator/pull/378/files#diff-075f4c8dc8c9bf09f4ff32ccfbf8b81cL23

Reply:
Weird, I wonder why we're not installing the latest Calico version for master